src/zenml/integrations/evidently/data_validators/evidently_data_validator.py

```diff
@@ -138,8 +140,23 @@ def _download_nltk_data() -> None:
         # Configure NLTK to use the current working directory to download and
         # lookup data. This is necessary because the default download directory
         # is not writable in some Docker containers.
+        os.environ["NLTK_DATA"] = os.getcwd()
         nltk_path.append(os.getcwd())
 
+    @staticmethod
+    def _download_nltk_data() -> None:
+        """Download NLTK data for text metrics and tests.
+
+        Raises:
+            ImportError: if NLTK is not installed.
+        """
+        try:
+            import nltk  # type: ignore[import-untyped]
+        except ImportError:
+            raise ImportError(
+                "NLTK is not installed. Please install NLTK to use "
```

Comment:
Doesn't nltk automatically get installed with Evidently anyway?

Reply:
for some reasons nltk is causing an issue with the download path, that code was legacy didn't change just moved it trying to force evidently download path